shared/package.json

```diff
@@ -95,6 +95,7 @@
     "emoji-datasource": "4.0.3",
     "emoji-datasource-apple": "4.0.3",
     "emoji-mart": "2.5.0",
+    "filesize": "^3.3.0",
```

Comment:
lets not add this dependency
also if you happen to install a dependency you must include the yarn.lock file

Reply:
This is not a new dependency though. I think another package we use already uses it. I just added it explicitly so if we happen to remove that package this won't break. Do we still not want this?

Reply:
if we use a package directly we must include it and make yarn.lock. even if some other package includes it it doesn't mean we want to . we want to reduce the packages we're including and this is a very simple function

Reply:
Hmm OK will remove it then.